pretext/SimplePythonData/OperatorsandOperands.ptx

```diff
@@ -14,79 +14,87 @@ hour * 60 + minute
 minute / 60
 5 ** 2
 (5 + 9) * (15 - 7)</pre>
-  <p>The tokens <c>+</c>,    <c>-</c>, and <c>*</c>, and the use of parenthesis for grouping,
-            mean in Python what they mean in mathematics. The asterisk (<c>*</c>) is the
-            token for multiplication, and <c>**</c> is the token for exponentiation.
-            Addition, subtraction, multiplication, and exponentiation all do what you
-            expect.</p>
+
+<p> Python use special <c> tokens </c> called <c>operators</c> that perform the following operations: </p>
+<ul> 
+<li> <c> + </c> addition </li>
+<li> <c> - </c> subtraction </li>
+<li> <c> * </c> multiplication </li>
+<li> <c> / </c> division </li>
+<li> <c> ** </c> exponentiation </li>
+<li> <c> () </c> grouping </li>
```

Comment:
I'm not sure if parenthesis is necessary; maybe we could at least rename it from "grouping" to be more like "mathematical parentheses"?

Reply:
I kept it as I didn't want to change the content (it was previosly there), I would prefer deleting it because its a topic of a future chapter (precedence)